GitUI/GitUICommands.cs

```diff
@@ -1706,7 +1706,7 @@ private string NormalizeFileName(string fileName)
         /// <returns>false on error.</returns>
         private bool RunFileHistoryCommand(IReadOnlyList<string> args, bool showBlame)
         {
-            string fileHistoryFileName = args[2];
+            string fileHistoryFileName = args[2].QuoteNE();
```

Comment:
I think this is too early here.
It should be done in `new FormBrowse(commands: this, new BrowseArguments` on line 1749 only.

Reply:
Done in the latest revision, though I had to call it also at line 1757 for the case when `FormFileHistory` is used instead of `FormBrowse`.